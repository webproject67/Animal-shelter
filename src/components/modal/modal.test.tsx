import * as React from "react";
import * as renderer from "react-test-renderer";
import Modal from "./modal";

const base = {
  name: 'Katrine',
  src: `1`
};

it(`Render Modal`, () => {
  const root = renderer.create(<Modal 
    base={base}
    onButtonClick={() => {}}
  />).toJSON();

  expect(root).toMatchSnapshot();
});