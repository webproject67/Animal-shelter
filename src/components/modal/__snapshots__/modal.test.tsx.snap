// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Modal 1`] = `
<div
  className="modal-content"
>
  <button
    aria-label="Ð¡ancel"
    className="modal-content__cancel"
    onClick={[Function]}
  >
    <svg
      className="modal-content__icon"
    >
      <use
        xlinkHref="img/svg/sprite.svg#cancel"
      />
    </svg>
  </button>
  <div
    className="modal-content__wrapper"
  >
    <img
      alt="Katrine"
      className="modal-content__img"
      src="img/desktop/1@1x.jpg"
      srcSet="img/desktop/1@2x.jpg 2x"
    />
    <div
      className="modal-content__description"
    >
      <p
        className="modal-content__name"
      >
        Katrine
      </p>
      <p
        className="modal-content__type1"
      >
        Pooch
      </p>
      <p
        className="modal-content__text"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce efficitur blandit condimentum. Proin accumsan lorem vel gra vida fringilla. Suspendisse potenti. Mauris ut pulvinar nunc. Donec consectetur, diam in porta tempus, urna ligula ves tibulum nibh.
      </p>
      <ul
        className="modal-content__list"
      >
        <li
          className="modal-content__item"
        >
          <p
            className="modal-content__type2"
          >
            Age: 
            <span
              className="modal-content__type2--color"
            >
              11
               months
            </span>
          </p>
        </li>
        <li
          className="modal-content__item"
        >
          <p
            className="modal-content__type2"
          >
            Inoculations: 
            <span
              className="modal-content__type2--color"
            >
              yes
            </span>
          </p>
        </li>
        <li
          className="modal-content__item"
        >
          <p
            className="modal-content__type2"
          >
            Diseases: 
            <span
              className="modal-content__type2--color"
            >
              yes
            </span>
          </p>
        </li>
        <li
          className="modal-content__item"
        >
          <p
            className="modal-content__type2"
          >
            Parasites: 
            <span
              className="modal-content__type2--color"
            >
              yes
            </span>
          </p>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
